cmd/convox/doctor.go

```diff
@@ -149,23 +149,24 @@ func cmdDoctor(c *cli.Context) error {
 	}
 
 	stdcli.Writef("\n\n### Build: Service\n")
-	startCheck("<file>docker-compose.yml</file> found")
-	_, err := os.Stat("docker-compose.yml")
+	dcm := helpers.DetectComposeFile()
```

Comment:
This is nitpicking, but perhaps we could have one `dcm = helpers.DetectComposeFile()` variable up in the `var()` scope.